tests/integration/test_fim/test_files/test_basic_usage/test_basic_usage_access_opened_files.py

```diff
@@ -0,0 +1,219 @@
+"""
+copyright: Copyright (C) 2015-2021, Wazuh Inc.
+
+           Created by Wazuh, Inc. <info@wazuh.com>.
+
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: Check that files that are being scanned by the syscheckd daemon can
+       modified (renamed/deleted), and that wazuh is not blocking the files.
+
+tier: 1
+
+modules:
+    - syscheck
+
+components:
+    - manager
+    - agent
+
+path: tests/integration/test_fim/test_files/test_basic_usage/test_basic_usage_access_opened_files.py
+
+daemons:
+    - wazuh-syscheckd
+
+modes:
+    - scheduled
+
+os_platform:
+    - linux
+    - windows
+    - solaris
+    - macos
+
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - CentOS 6
+    - Ubuntu Focal
+    - Ubuntu Bionic
+    - Ubuntu Xenial
+    - Ubuntu Trusty
+    - Debian Buster
+    - Debian Stretch
+    - Debian Jessie
+    - Debian Wheezy
+    - Red Hat 8
+    - Red Hat 7
+    - Red Hat 6
+    - Windows 10
+
+references:
+    - https://documentation.wazuh.com/current/user-manual/capabilities/file-integrity/fim-configuration.html
+    - https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/localfile.html
+
+tags:
+    - syscheck
+"""
+
+import os
+import random
+import string
+import pytest
+import time
+
+from wazuh_testing.fim import LOG_FILE_PATH, generate_params, detect_initial_scan_start, get_scan_timestamp
+from wazuh_testing.tools import PREFIX
+from wazuh_testing.tools.configuration import load_wazuh_configurations, check_apply_test
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.tools.file import delete_path_recursively, create_large_file
+
+
+# Marks
+pytestmark = [pytest.mark.tier(level=1)]
+
+# Variables
+directory_str = os.path.join(PREFIX, "testdir1")
+file_path = os.path.join(directory_str, "large_file")
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "data")
+configurations_path = os.path.join(test_data_path, "wazuh_conf.yaml")
+
+# configurations
+conf_params = {"TEST_DIRECTORIES": directory_str, "MODULE_NAME": __name__}
+parameters, metadata = generate_params(extra_params=conf_params, modes=['scheduled'])
+configurations = load_wazuh_configurations(configurations_path, __name__, params=parameters, metadata=metadata)
+
+
+# Fixtures
+@pytest.fixture(scope="module", params=configurations)
+def get_configuration(request):
+    """Get configurations from the module."""
+    return request.param
+
+
+@pytest.fixture(scope="function")
+def create_and_delete_file(request):
+    """Creates a file and later deletes the path."""
+
+    create_large_file(directory_str, file_path)
+    yield
+    delete_path_recursively(directory_str)
```

Comment:
The method to generate the file is in `deps/tools/files.py`, so it can be reused. 
Here it is used as part of a fixture, that after the test is done, makes sure to delete the path where the file was created.